components/renku_data_services/session/api.spec.yaml

```diff
@@ -301,41 +301,49 @@ components:
         - uid
         - gid
         - default_url
-      example:
-        id: 01AN4Z79ZS6XX96588FDX0H099
-        name: JupyterLab environment
-        creation_date: "2023-11-01T17:32:28Z"
-        description: JupyterLab session environment
-        container_image: renku-jupyter:latest
-        default_url: "/lab"
-        port: 8080
-        working_directory: /home/jovyan/work
-        mount_directory: /home/jovyan/work
-        uid: 1000
-        gid: 1000
-        is_archive: false
     EnvironmentGetInLauncher:
       allOf:
-        - $ref: "#/components/schemas/Environment"
+        - oneOf:
+          - $ref: "#/components/schemas/EnvironmentWithImageGet"
+          - $ref: "#/components/schemas/EnvironmentWithBuildGet"
         - type: object
           properties:
             environment_kind:
               $ref: "#/components/schemas/EnvironmentKind"
           required:
             - environment_kind
-          example:
-            environment_kind: global_environment
+    EnvironmentWithImageGet:
+      allOf:
+        - $ref: "#/components/schemas/Environment"
+        - type: object
+          properties:
+            environment_image_source:
+              $ref: "#/components/schemas/EnvironmentImageSourceImage"
+          required:
+            - environment_image_source
+    EnvironmentWithBuildGet:
+      allOf:
+        - $ref: "#/components/schemas/Environment"
```

Comment:
This has `container_image` as required, non-nullable with a regex pattern. What value is this going to be before the first image is built?

Reply:
Good point! I've made it optional in this case.